h/models/group.py

```diff
@@ -157,6 +157,8 @@ class Group(Base, mixins.Timestamps):
         sa.Enum(WriteableBy, name="group_writeable_by"), nullable=True
     )
 
+    pre_moderated: Mapped[bool | None] = mapped_column()
```

Comment:
Made this nullable to simplify deployment.

I added a note to the board to consider if we want to run a migration to make all nulls here false.

Reply:
The good thing is that `None` is a "falsy" value, and we probably want this to be false by default.